tests/integration/test_keeper_znode_time/test.py

```diff
@@ -134,9 +134,9 @@ def test_server_restart(started_cluster):
         node2_zk = get_fake_zk("node2")
         node3_zk = get_fake_zk("node3")
         for child_node in range(1000):
-            stats1 = node1_zk.exists("/test_between_servers/" + str(child_node))
-            stats2 = node2_zk.exists("/test_between_servers/" + str(child_node))
-            stats3 = node3_zk.exists("/test_between_servers/" + str(child_node))
+            stats1 = node1_zk.exists("/test_server_restart/" + str(child_node))
```

Comment:
It's interesting how this caused problems for rerun because in the initial run it was being run after the `test_between_servers` which created the nodes, and each retry run the test solo causing the nodes to be missing.
The problematic part is how it failed during the initial run and it seems really random.